test/distributed/test_c10d_nccl.py

```diff
@@ -353,27 +350,25 @@ def allreduce(tensors, op):
 
                     allreduce(tensors, c10d._make_nccl_premul_sum(factor))
 
-                    # TODO(#38095): Replace assertEqualIgnoreType. See issue #38095
-                    self.assertEqualIgnoreType(
-                        factor * torch.tensor([float(self.world_size * (self.world_size + 1) / 2)], device=local_device_id),
+                    self.assertEqual(
+                        factor * torch.tensor([self.world_size * (self.world_size + 1) / 2],
```

Comment:
Curious: Are you using an auto formatter to achieve this indentation?

Reply:
I this case I just manually formatted it.